Client/mods/deathmatch/logic/CClientVehicle.cpp

```diff
@@ -1472,6 +1472,24 @@ void CClientVehicle::SetWheelStatus(unsigned char ucWheel, unsigned char ucStatu
     }
 }
 
+//
+// Returns component name for eWheelPosition enum
+//
+const SString CClientVehicle::GetComponentNameForWheel(unsigned char ucWheel) const noexcept
```

Comment:
It is strongly advised to return values as non-const

Reply:
I don't know who to listen to. TheNormalnij wrote that I should give const. You write that it would be better not to const. I don't want to keep playing like this because everyone has a different opinion.